deltachat-ffi/deltachat.h

```diff
@@ -1037,14 +1037,18 @@ int dc_send_webxdc_status_update (dc_context_t* context, uint32_t msg_id, const
  * @memberof dc_context_t
  * @param context The context object
  * @param msg_id id of the message with the webxdc instance
- * @param status_update_id Can be used to filter out only a concrete status update.
- *     When set to 0, all known status updates are returned.
- * @return JSON-array containing the requested updates,
- *     each element was created by dc_send_webxdc_status_update()
- *     on this or other devices.
- *     If there are no updates, an empty JSON-array is returned.
+ * @param serial The last known serial, defaults to 0.
```

Comment:
How does it "default to" in C? "Pass 0 if there are no known serials to receive all updates" or something like this is more clear if I understand it correctly.

Reply:
thanks, you are right. i have changed that.